packages/neherlab-covid-19-scenarios/script/bootstrap

```diff
@@ -6,30 +6,34 @@ set -e
 
 if [ ! -d $BASEL_REPO ]; then
 cat <<MESSAGE
-Set the 'BASEL_REPO' to the path to the Basel model repository,
+The Basel model repository is at https://github.com/neherlab/covid19_scenarios.
+Set the 'BASEL_REPO' environment variable to the checkout path of the model repository
 or check out the repository to ../../../covid19_scenarios
-For now we recommend you check out this fork and branch:
-https://github.com/adityasharad/covid19_scenarios/tree/cli-entry-point
 MESSAGE
 exit 1
 fi
 
-# Build the Basel repo and create an executable
-rm -rf .local/bin
-mkdir -p .local/bin
-CURRENT=$PWD
-cd $BASEL_REPO
-echo "Compiling the Basel model CLI entry point"
+# Build the Basel model schema
+pushd $BASEL_REPO
+echo "Installing Basel model dependencies"
+yarn install --frozen-lockfile --ignore-scripts
```

Comment:
Unfortunate that you have to run with `ignore-scripts`. Is it because of this? https://www.npmjs.com/package/snyk Which is interesting and something I've never heard of until now.

Reply:
Exactly. This froze up while it was trying to talk to Snyk.

Reply:
Hmmm...wonder if they should try running a _different_ product that looks for vulnerabilities in your code. :)